src/crossChain/AxelarProvider.ts

```diff
@@ -38,6 +44,12 @@ export class AxelarProvider implements CrossChainProvider {
   async init() {
     if (!this.squid.initialized) {
       await this.squid.init()
+      const polygonChainData = this.squid.chains.find(
+        c => c.chainId === ChainId.MATIC_MAINNET.toString()
+      )
+      if (polygonChainData?.squidContracts.squidMulticall) {
+        this.squidMulticall = polygonChainData.squidContracts.squidMulticall
```

Comment:
Does the multi call contract address changes according to the chain the user wants to send the transaction to?